internal/exec/exec.go

```diff
@@ -554,7 +557,9 @@ func (e *fieldExec) execField(ctx context.Context, r *request, f *query.Field, r
 	result, err := e.execField2(spanCtx, r, f, resolver, span)
 
 	if err != nil {
-		r.addError(errors.Errorf("%s", err))
+		queryError := errors.Errorf("%s", err)
+		queryError.ResolverError = err
+		r.addError(queryError)
 		addResult(f.Alias, nil) // TODO handle non-nil
```

Comment:
This may be a discussion for another thread but shouldn't this be `addResult(f.Alias, result)` if the handler was able to resolve a partial amount of data, shouldn't we still send that to client and allow them to figure out what to do with it?